src/OpenTK.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -0,0 +1,37 @@
+using System.Drawing;
+using Humanizer;
+using Humanizer.Localisation.Formatters;
+using OpenToolkit.GraphicsLibraryFramework;
+using OpenToolkit.Windowing.Common;
+
+namespace OpenToolkit.Windowing.Desktop
+{
+    public class NativeWindowSettings : INativeWindowProperties
```

Comment:
reason why it's called NativeWindowSettings and not NativeWindowProperties?
Same for GameWindowSettings : IGameWindowProperties

Reply:
Just ended up being the name we went with. If GameWindowProperties would make more sense, we could do that instead.